app/constants/teacherStyle.ts

```
@@ -0,0 +1,19 @@
+import solar_fire from "../../public/images/solar_fire.png";
```

ESLint

[error] Identifier 'solar_fire' is not in camel case.

(camelcase)